src/components/sidebar/sidebar.css

```diff
@@ -0,0 +1,96 @@
+
+
+.product-main-sidebar{
+    display: grid;
+    height: 100vh;
+    grid-template-columns: 20rem 1fr;
+}
+
+.product-sidebar{
+    grid-column: 1 / 2;
+}
+
+.filter{
+    display: flex;
+    justify-content:space-between;
+    align-items: center;
+    height: 5rem;
+}
+
+.product-main{
+    grid-column: 2 / 3;
+    margin-top: 10rem;
+}
+
+.product-cards{
+    border-radius: 0rem;
```

Comment:
we need to use component library in three of the projects, if you are planning to use later then its okay.

Reply:
my cards and navbar is coming from component library this line is just for testing something,forget to remove